docs/source/applications.rst

```diff
@@ -7,25 +7,28 @@ Applications
 Introduction
 ============
 
-Applications are python scripts built on the :ref:`Library` that execute a simple, well defined task.
-These applications will be used as building blocks for the Simulation System Workflows.
+Applications are python scripts built on the :ref:`Library` that execute a simple, well defined
+task. These applications will be used as building blocks for the Simulation System Workflows.
 
 The application scripts can be found in gammasim-tools/applications.
 
-The applications expect a config file named config.yml in your gammasim-tools directory (unless specified differently
-using the function function config.setConfigFileName) . Please,
-follow the instructions in :ref:`Configuration` to setup your config file.
+The applications expect a config file named config.yml in your gammasim-tools directory (unless
```

Comment:
This is a comment to myself that this paragraph needs an update given the new Configuration class